app/src/main/java/com/eduard/caloriecounter/presentation/presenter/CollectionPresenter.java

```diff
@@ -0,0 +1,71 @@
+package com.eduard.caloriecounter.presentation.presenter;
+
+import android.content.DialogInterface;
+import android.view.View;
+import android.widget.Button;
+
+import com.eduard.caloriecounter.presentation.base.BasePresenter;
+import com.eduard.caloriecounter.presentation.model.User;
+import com.eduard.caloriecounter.presentation.utils.Calculator;
+
+public class CollectionPresenter extends BasePresenter<CollectionContract.View> implements CollectionContract.Presenter {
+
+    private User mUser;
+    private Calculator mCalculator;
+
+    private User mWeight;
+    private User mHeight;
+    private User mAge;
+
+    private boolean isInErrorState;
+
+    public CollectionPresenter(Calculator calculator,User user) {
+
+        mCalculator=calculator;
+        mUser=user;
+    }
+
+//    @Override
+//    public void onClick(android.view.View view) {
+////        float data = user.getInfo();
+////        getView().updateUserInfo(data);
+//    }
+
+    @Override
+    public void appendValue(String value) {
+
+    }
+
+    @Override
+    public void performCalculation() {
+        String result = "";
+
+        result = mCalculator.operation(mWeight,mHeight,mAge);
+
```

Comment:
Too much spaces, why you are not formatting your code?

Reply:
ready